ci/resourcetypes.rego

```diff
@@ -0,0 +1,27 @@
+package aws.cloudformation
+
+import future.keywords
+
+# METADATA
+# description: |
+#   Get all resource types from the single-file JSON schema provided by AWS for the us-west-1
```

Comment:
`us-west-1`'s missing hundreds of resource types (byfar the least supported `us` region)
`us-east-1` is the de facto AWS standard: https://github.com/patmyron/cloud

I'll send over an updated version of https://github.com/iann0036/cfn-hooks/pull/1

Reply:
Thanks @PatMyron! Much appreciated 👍